### PR TITLE
feat: viewer-crd-controller rock integration

### PR DESCRIPTION
**Pull request by i-chvets**
Some minor adjustments are required since last time this ROCK was implementd. See comments below.

Summary of changes:
- Updated version.
- Added ne non-root user setup.
- Removed old non-root user setup.
- Updated security requirement to use proper option.

### CODE REVIEW

File: viewer-crd-controller/rockcraft.yaml [hunk 1]
```diff
@@ -42,13 +41,13 @@ parts:
...
       # security requirement
-      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/ROCK images
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
       (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
-      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
-      > ${CRAFT_PART_INSTALL}/usr/share/ROCK images/dpkg.query
+      dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
 
   non-root-user:
```
Comment (DnPlas) [inline viewer-crd-controller/rockcraft.yaml:52]: In other PRs you removed the `non-root-user` part, should we also do that here?

**Review by DnPlas — COMMENTED** [review]
Small comment, thanks @i-chvets

File: viewer-crd-controller/rockcraft.yaml (hunk 1 above)
Comment (i-chvets) [reply viewer-crd-controller/rockcraft.yaml:52]: Yes. It needs to be removed.

Comment (i-chvets) [reply viewer-crd-controller/rockcraft.yaml]: Done.

**Review by DnPlas — APPROVED** [review]

**Pull request merged**